tasks/main.yml

```diff
@@ -2,13 +2,17 @@
 ---
 # Set up internal variables.
 - name: Set version specific variables
-  include_vars: "{{ item }}"
-  with_first_found:
-    - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
-    - "{{ ansible_distribution }}.yml"
-    - "{{ ansible_os_family }}.yml"
-    - "default.yml"
+  include_vars: "{{ lookup('first_found', ffparams) }}"
+  vars:
+    ffparams:
+      files:
+        - "{{ ansible_facts.distribution }}_{{ ansible_facts.distribution_version }}.yml" # yamllint disable-line rule:line-length
+        - "{{ ansible_facts.distribution }}_{{ ansible_facts.distribution_major_version }}.yml" # yamllint disable-line rule:line-length
+        - "{{ ansible_facts.distribution }}.yml"
+        - "{{ ansible_facts.os_family }}.yml"
```

Comment:
If you are disabling yamllint checking the line length, then you can revert back to the bracket form `ansible_facts['distribution']` ...

Reply:
Is there a reason to prefer the bracket notation? I find the dot notation more readable (and also shorter).

Reply:
> Is there a reason to prefer the bracket notation? I find the dot notation more readable (and also shorter).

https://github.com/oasis-roles/meta_standards#supporting-multiple-distributions-and-versions
"... the bracket notation is what is used in Ansible documentation such as Commonly Used Facts and Operating System and Distribution Variance"

It doesn't matter to me - according to jinja2 they are practically equivalent